python/setup.py

```diff
@@ -16,14 +16,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import importlib.util
 import glob
 import os
 import sys
 from setuptools import setup
+from setuptools.command.install import install
 from shutil import copyfile, copytree, rmtree
 
 try:
     exec(open('pyspark/version.py').read())
+    spec = importlib.util.spec_from_file_location("install", "pyspark/install.py")
+    install_module = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(install_module)
 except IOError:
     print("Failed to load PySpark version file for packaging. You must be in Spark's python dir.",
```

Comment:
When we do packaging, we also need exec `install_module`? Or we need to update this message?